test/HeckeMiscMatrix-test.jl

```diff
@@ -0,0 +1,42 @@
+@testset "Spectrum and eigenspaces" begin
+  K = GF(5)
+  M = matrix(K, 4, 4, [ 0, 0, 1, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 1 ])
+  l = eigenvalues(M)
+  @test length(l) == 1
+  @test l[1] == one(K)
+  l_m = eigenvalues_with_multiplicities(M)
+  @test length(l_m) == 1
+  @test l_m[1] == (one(K), 2)
+
+  L = GF(5, 2)
+  a = 2*gen(L) + 1 # third root of 1
+  lL = eigenvalues(L, M)
+  @test length(lL) == 3
+  @test one(L) in lL && a in lL && a^-1 in lL
```

Comment:
```suggestion
  @test one(L) in lL
  @test a in lL
  @test a^-1 in lL
```
this makes it easier if a test fails to find the failing test